### PR TITLE
Fix double slash in basepath usage

### PR DESCRIPTION
**Pull request by freekrai**

### CODE REVIEW

**Review by rijkvanzanten — APPROVED** [review]

File: gatsby-node.js [hunk 1]
```diff
@@ -150,7 +150,9 @@ class Plugin {
 
 		try {
 			const baseUrl = new URL(url);
-			const basePath = baseUrl.pathname;
+			let basePath = baseUrl.pathname;
+
+			basePath.endsWith('/') ? basePath.slice(0, -1) : basePath;
```
Comment (rijkvanzanten) [inline gatsby-node.js]: this doesn't exec currently

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: gatsby-node.js (hunk 1 above)
Comment (freekrai) [reply gatsby-node.js]: yeah, just pushed the fix.

**Review by licitdev — COMMENTED** [review]
I wonder if the same logic could be more concise 🤔 

For example: 
```ts
if(baseUrl.pathname.endsWith('/')) {
  baseUrl.pathname =  baseUrl.pathname.slice(0, -1);
}
```

File: gatsby-node.js [hunk 2]
```diff
@@ -150,7 +150,9 @@ class Plugin {
 
 		try {
 			const baseUrl = new URL(url);
-			const basePath = baseUrl.pathname;
+			let basePath = baseUrl.pathname;
+
+			basePath = basePath.endsWith('/') ? basePath.slice(0, -1) : basePath;
```
Comment (rijkvanzanten) [inline gatsby-node.js]: as per @licitdev's comment:

```suggestion
			const basePath = baseUrl.pathname.endsWith('/') ? baseUrl.pathname.slice(0, -1) : baseUrl.pathname;
```

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**